src/MobileApps/MyDriving/MyDriving/ViewModel/CurrentTripViewModel.cs

```diff
@@ -194,9 +194,14 @@ public string EngineLoad
             if (IsRecording)
                 return false;
 
-            var track = Logger.Instance.TrackTime("SaveRecording");
```

Comment:
Yes. But it does not do harm to keep them and enable them when HockeyApp supports it?